beaupy/_beaupy.py

```diff
@@ -231,6 +231,7 @@ def prompt(
 
 def select(
     options: List[Union[Tuple[int, ...], str]],
+    title: Optional[str] = None,
```

Comment:
As is, this constitutes a breaking change as anyone using `beaupy` without named arguments and relying on argument order would encounter issues. 

We could avoid a breaking change if the `title` was made the last argument, but that's a not very natural place for it to be. I am not sure if it is worth a breaking change and associated major-version bump. Wdyt?

Reply:
To clarify, I am not against breaking changes and I think this one might be worthwhile. If you agree, I am going to branch out master into a new branch for the `4.0.0` version, as there are more things I want to wiggle around in the API and we should make the pull request against said new branch.

Reply:
Got it. I definitely agree that it's a too small feature for a major bump, and that the natural place for it would be one of the first args. I'd say it could stay on hold in a "staging" branch, and be released along with other changes on a next major bump. I can help by working on the original purpose of https://github.com/petereon/beaupy/issues/74 and other issues too.

Reply:
Or it could maybe be released as b0 of next major version

Reply:
Sounds great! I've made a staging branch for eventual `v4`,  [staging/v4.0](https://github.com/petereon/beaupy/tree/staging/v4.0). Could you please move this pull request to that branch?

As for the other issues, I will gladly accept contributions!

Reply:
Just noticed the type of title is wrong in the comments, will update it quickly

Reply:
Done ✅